src/Project/JobRunner.ts

```diff
@@ -98,6 +98,9 @@ export class JobRunner extends EventEmitter {
 
   private onCleanup() {
     this.running = false;
+    if (process.env.userp !== '') {
```

Comment:
(Optional) This looks okay when `if condition` is removed even.

Reply:
Yes, right. I just want to express it more accurately. :) I will update it.